docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -384,11 +368,11 @@ zk-0      0/1       Terminating   0         11m
 Reapply the manifest in `zookeeper.yaml`.
 
 ```shell
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/website/master/docs/tutorials/stateful-application/zookeeper.yaml
+kubectl apply -f https://k8s.io/docs/tutorials/stateful-application/zookeeper.yaml
 ```
 
-The `zk` StatefulSet will be created, but, as they already exist, the other API 
-Objects in the manifest will not be modified.
+This creates the `zk` StatefulSet, but, as they already exist, the other API
+Objects in the manifest are not modified.
```

Comment:
Maybe clear this one up even a bit more? Something like:
`This creates the `zk` StatefulSet object, but the other API objects in the manifest are not modified because they already exist.`
(`they already exist` is confusingly placed in the original ...)